woocommerce/woocommerce-gutenberg-products-block
# Allow extensions to filter Total label

Part of #3705.

### Screenshots

https://user-images.githubusercontent.com/3616980/105330750-f1a03400-5bd2-11eb-8e42-99c9364c163e.mp4

### How to test the changes in this Pull Request:

1. Check out this branch in WC Subscriptions: `add/wc-blocks-filter-totals-label` (see associated PR: 3965-gh-woocommerce/woocommerce-subscriptions).
2. Add a subscription product to your cart + any other product which is not a subscription.
3. Go to the Cart or Checkout blocks and verify the sidebar `Total` label is now `Total due today`.
4. Remove the subscription from your cart and verify the `Total due today` label changes to `Total`.
